checkov/terraform/checks/graph_checks/gcp/GCPComputeFirewallOverlyPermissiveToAllTraffic.yaml

```
@@ -0,0 +1,40 @@
+metadata:
+  name: "Ensure GCP compute firewall ingress does not allow unrestricted access to all ports"
+  id: "CKV2_GCP_12"
+  guidelines: "This policy identifies GCP Firewall rules which allows inbound traffic on all protocols through the public internet. Doing so, may allow a bad actor to brute force their way into the system and potentially get access to the entire network."
+  category: "networking"
+  severity: "high"
```

```suggestion
  category: "NETWORKING"
```
no need for guideline and severity here 🙂